qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -133,6 +133,41 @@
     }
   }
 
+
+  /*
+   *
+   * Updates deprecated flag in the prep information <fieldset>
+   *
+   * @param prep_id the prep template id
+   *
+   * This function executes an AJAX call to update the deprecated flag
+   *
+   */
+  function deprecate_preparation(prep_id, new_value) {
+    var status = 'deprecate';
+    console.log(new_value)
+    if (!new_value){
+      status = 'DE-deprecate'
+    }
+    if(confirm("Are you sure you want to " + status + " this preparation?")) {
+      if(confirm("Are you _really_ sure you want to " + status + " this preparation?")) {
```

Comment:
Two "alerts" seems like a little too much in my opinion. Is there a strong motivation for this?

Reply:
Just that in sensitive things we always add two but in this case it's pretty easy to correct so I will leave only one.

Reply:
My bad, if two confirmations are widespread in Qiita then I rather be consistent than go for a personal preference. Thanks eitherway! 👍